js/src/personal/riyuan/components/experience/tabitem.tsx

```diff
@@ -0,0 +1,112 @@
+import { List, Tabs, Text } from '@mantine/core'
```

Comment:
Fix this file's name. Also this file has everything for tabs? TabItem.tsx might not make snse.